server/src/test/java/org/elasticsearch/search/fetch/StoredFieldsSpecTests.java

```diff
@@ -69,6 +71,8 @@ public void testScriptFieldsEnableMetadata() {
     private static SearchContext searchContext(SearchSourceBuilder sourceBuilder) {
         SearchContext sc = mock(SearchContext.class);
         when(sc.fetchSourceContext()).thenReturn(sourceBuilder.fetchSource());
+        when(sc.getSearchExecutionContext()).thenReturn(mock(SearchExecutionContext.class));
+        when(sc.getSearchExecutionContext().getMappingLookup()).thenReturn(MappingLookup.EMPTY);
```

Comment:
I think you want to:

- Create a mock `SearchExecutionContext` first
- Configure mock object to return an empty mapping lookup
- Configure the mock `SearchContext` to return the configured mock `SearchExecutionContext`